src/Cronos/CronParser.cs

```diff
@@ -280,6 +308,23 @@ private static unsafe ulong ParseStar(CronField field, ref char* pointer)
                 : field.AllBits;
         }
 
+        private static unsafe ulong ParseHash(CronField field, ref char* pointer, Random? rng)
+        {
+            if (rng == null) throw new ArgumentNullException(nameof(rng), "Using H in the format requires providing a jitter seed");
```

Comment:
Nullable reference type feature is enabled for the project, so such exceptions can be avoided by using `Random` instead of `Random?` data type.